scripts/interface/lint.mjs

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env zx
+import 'zx/globals';
+import {
+  cliArguments,
+  getToolchainArgument,
+  popArgument,
+  workingDirectory,
+} from '../utils.mjs';
+
+// Configure additional arguments here, e.g.:
+// ['--arg1', '--arg2', ...cliArguments()]
+const lintArgs = cliArguments();
+
+const fix = popArgument(lintArgs, '--fix');
+const toolchain = getToolchainArgument('lint');
+const manifestPath = path.join(workingDirectory, 'interface', 'Cargo.toml');
+const args = [
+  '--features',
+  'frozen-abi',
+  ...lintArgs,
+  '--',
+  '--deny=warnings',
+  '--deny=clippy::default_trait_access',
+  '--deny=clippy::arithmetic_side_effects',
+  '--deny=clippy::manual_let_else',
+  '--deny=clippy::used_underscore_binding',
+];
```

Comment:
These should be under `lintArgs` (above), so the provided `cliArguments()` can be chained to them.

Reply:
I think we need to do something like this:
```ts
const lintArgs = ['--features', 'frozen-abi', ...cliArguments()];
// Check whether a '--' was already used.
if (lintArgs.indexOf('--') === -1) {
  lintArgs.push('--');
}
// Add additional arguments.
lintArgs.push(
  '--deny=warnings',
  '--deny=clippy::default_trait_access',
  '--deny=clippy::arithmetic_side_effects',
  '--deny=clippy::manual_let_else',
  '--deny=clippy::used_underscore_binding'
);
```

Reply:
Ohh, in the other programs we've just appended `cliArgs()` on the backside of the `--`. You want the user to be able to provide cargo args, too?

Reply:
The difference here is that we are providing some "default" cargo args, so there might be some specified on the command-line already.